app/gui/src/main/scala/org/bitcoins/gui/WalletGUIModel.scala

```diff
@@ -151,4 +154,25 @@ class WalletGUIModel(dlcModel: DLCPaneModel)(implicit system: ActorSystem) {
         }.showAndWait()
     }
   }
+
+  private def updateWalletAccounting(): Unit = {
+    ConsoleCli.exec(GetDLCWalletAccounting, GlobalData.consoleCliConfig) match {
+      case Failure(err) =>
+        err.printStackTrace()
+        val _ = new Alert(AlertType.Error) {
+          initOwner(owner)
+          title = "Could not retrieve dlc wallet accounting"
+          headerText = s"Operation failed. Exception: ${err.getClass}"
+          contentText = err.getMessage
+        }.showAndWait()
```

Comment:
We shouldn't create an alert here, this could get really annoying where every 10 seconds an alert pops up with an error.

We should probably just log the error and continue.

Maybe eventually put up a warning `!` next to the text with a tooltip saying there's an error with the wallet accounting.